features/checkout/skipping_shipping_step_when_order_does_not_require_any_shipping.feature

```diff
@@ -24,6 +25,7 @@ Feature: Skipping shipping step when order does not require any shipping
     Scenario: Seeing checkout shipping page after addressing if at least one of order items require shipping
         Given I have "Guards! Guards! - ebook" variant of product "Guards! Guards!" in the cart
         And I have "Guards! Guards! - book" variant of product "Guards! Guards!" in the cart
+        Then my cart shipping total should be "$5.00"
```

Comment:
The flow of the Behat scenario should be: `Given` -> `When` -> `Then` (with some `And`s between them). Therefore, we should not check the result of some assumption and then make more assumptions for further check. The best choice would be to create two new scenarios that would just check cart shipping total based one some `Given`s. Like `Seeing free shipping total when none of order items require shipping` and `Seeing not free shipping total when at least one of order items require shipping`? They could be even places in separate feature, I think ;)